### PR TITLE
change: [M3-8362] – Update placement group policy text copy

### PR DESCRIPTION
**Pull request by carrillo-erik**
## Description 📝
This PR changes the text copy found in the Placement Group create drawer, under the "Placement Group Policy" radio buttons descriptions.

## Changes  🔄
The text copy updates are as follows:
- Strict
  - **Old**: You can’t assign Linodes if the preferred container defined by your Affinity Type lacks capacity or is unavailable (best practice).
  - **New**: Allows the addition of more compute instances to the group as long as the placement group remains compliant.
- Flexible
  - **Old**: You can assign Linodes, even if they’re not in the preferred container defined by your Affinity Type, but your placement group will be non-compliant.
  - **New**: Allows the addition of more compute instances to the group even if it breaks the placement group’s compliance.
- Created two constants to hold the text copy instead of hard coding it inside the component.

## Target release date 🗓️
08/05/2024

## Preview 📷
| Before  | After   |
| ------- | ------- |
| old-pg-copy | new-pg-copy |

### Prerequisites
(How to setup test environment)
- Pull this branch and run it on your local machine.
- Visit the Placement Groups create drawer by using the `Create` dropdown menu or the Placement Groups landing page and clicking the `Create Placement Group` button.

### Verification steps
(How to verify changes)
- Verify that the copy updates reflected in these changes match the description found in the `Changes` section above.

## As an Author I have considered 🤔
*Check all that apply*

- [x] 👀 Doing a self review
- [ ] ❔ Our [contribution guidelines](https://github.com/linode/manager/blob/develop/docs/CONTRIBUTING.md)
- [ ] 🤏 Splitting feature into small PRs
- [x] ➕ Adding a changeset
- [ ] 🧪 Providing/Improving test coverage
- [ ] 🔐 Removing all sensitive information from the code and PR description
- [ ] 🚩 Using a feature flag to protect the release
- [ ] 👣 Providing comprehensive reproduction steps
- [ ] 📑 Providing or updating our documentation
- [ ] 🕛 Scheduling a pair reviewing session
- [ ] 📱 Providing mobile support
- [ ] ♿  Providing accessibility support

### CODE REVIEW

**Review by dwiley-akamai — APPROVED** [review]
Confirmed copy changes ✅

Should we have a changeset for this?

**Comment by carrillo-erik** [comment]
> Confirmed copy changes ✅
> 
> Should we have a changeset for this?

I just pushed it, I had checked the checkbox but got sidetracked and hadn't included it.

**Review by AzureLatte — APPROVED** [review]

**Comment by jdamore-linode** [comment]
@carrillo-erik Merging in the latest changes from `develop` should resolve the Volumes failures, and the Linode delete failure can be disregarded because it's been flaky for a while, but I'm hoping to get to the bottom of those failures in the Linode Create w/ Firewall tests

File: packages/manager/cypress/e2e/core/linodes/create-linode-with-firewall.spec.ts [hunk 1]
```diff
@@ -32,6 +32,10 @@ describe('Create Linode with Firewall', () => {
    * - Confirms that outgoing Linode Create API request specifies the selected Firewall to be attached.
    */
   it('can assign existing Firewall during Linode Create flow', () => {
+    mockAppendFeatureFlags({
+      gecko2: makeFeatureFlagData(false),
+    }).as('getFeatureFlags');
+
```
Comment (jdamore-linode) [inline packages/manager/cypress/e2e/core/linodes/create-linode-with-firewall.spec.ts:38]:
```suggestion
```

File: packages/manager/cypress/e2e/core/linodes/create-linode-with-firewall.spec.ts [hunk 2]
```diff
@@ -110,6 +114,9 @@ describe('Create Linode with Firewall', () => {
       region: linodeRegion.id,
     });
 
+    mockAppendFeatureFlags({
+      gecko2: makeFeatureFlagData(false),
+    }).as('getFeatureFlags');
```
Comment (jdamore-linode) [inline packages/manager/cypress/e2e/core/linodes/create-linode-with-firewall.spec.ts:119]:
```suggestion
```

File: packages/manager/cypress/e2e/core/volumes/create-volume.spec.ts [hunk 3]
```diff
@@ -27,6 +29,10 @@ describe('volume create flow', () => {
    * - Confirms that volume is listed correctly on volumes landing page.
    */
   it('creates an unattached volume', () => {
+    mockAppendFeatureFlags({
+      gecko2: makeFeatureFlagData(false),
+    }).as('getFeatureFlags');
+
```
Comment (jdamore-linode) [inline packages/manager/cypress/e2e/core/volumes/create-volume.spec.ts:35]:
```suggestion
```

File: packages/manager/cypress/e2e/core/volumes/create-volume.spec.ts [hunk 4]
```diff
@@ -72,6 +78,10 @@ describe('volume create flow', () => {
    * - Confirms that volume is listed correctly on Linode 'Storage' details page.
    */
   it('creates an attached volume', () => {
+    mockAppendFeatureFlags({
+      gecko2: makeFeatureFlagData(false),
+    }).as('getFeatureFlags');
+
```
Comment (jdamore-linode) [inline packages/manager/cypress/e2e/core/volumes/create-volume.spec.ts:84]:
```suggestion
```

File: packages/manager/cypress/e2e/core/volumes/create-volume.spec.ts [hunk 5]
```diff
@@ -153,6 +163,10 @@ describe('volume create flow', () => {
       booted: false,
     });
 
+    mockAppendFeatureFlags({
+      gecko2: makeFeatureFlagData(false),
+    }).as('getFeatureFlags');
+
```
Comment (jdamore-linode) [inline packages/manager/cypress/e2e/core/volumes/create-volume.spec.ts:169]:
```suggestion
```

**Review by jdamore-linode — CHANGES_REQUESTED** [review]
Looks like the changes in `create-linode-with-firewall.spec.ts` are causing the test failures in CI. Calling `mockAppendFeatureFlags()` in the test body overrides the call to `mockAppendFeatureFlags()` in the before hook, causing the tests to fail in any LD environment where the Linode Create refactor is not actually enabled.

Normally in a case like this, the solution would be to consolidate the calls to `mockAppendFeatureFlags()`, but in this case we'll just remove all of these changes since the `gecko2` feature flag override isn't necessary since the related PR was reverted.

File: packages/manager/cypress/e2e/core/linodes/create-linode-with-firewall.spec.ts (hunk 1 above)
Comment (carrillo-erik) [reply packages/manager/cypress/e2e/core/linodes/create-linode-with-firewall.spec.ts]: Removed.

File: packages/manager/cypress/e2e/core/linodes/create-linode-with-firewall.spec.ts (hunk 2 above)
Comment (carrillo-erik) [reply packages/manager/cypress/e2e/core/linodes/create-linode-with-firewall.spec.ts]: Removed.

File: packages/manager/cypress/e2e/core/volumes/create-volume.spec.ts (hunk 3 above)
Comment (carrillo-erik) [reply packages/manager/cypress/e2e/core/volumes/create-volume.spec.ts]: Removed.

File: packages/manager/cypress/e2e/core/volumes/create-volume.spec.ts (hunk 4 above)
Comment (carrillo-erik) [reply packages/manager/cypress/e2e/core/volumes/create-volume.spec.ts]: Removed.

File: packages/manager/cypress/e2e/core/volumes/create-volume.spec.ts (hunk 5 above)
Comment (carrillo-erik) [reply packages/manager/cypress/e2e/core/volumes/create-volume.spec.ts]: Removed.

**Comment by carrillo-erik** [comment]
> Looks like the changes in `create-linode-with-firewall.spec.ts` are causing the test failures in CI. Calling `mockAppendFeatureFlags()` in the test body overrides the call to `mockAppendFeatureFlags()` in the before hook, causing the tests to fail in any LD environment where the Linode Create refactor is not actually enabled.
> 
> Normally in a case like this, the solution would be to consolidate the calls to `mockAppendFeatureFlags()`, but in this case we'll just remove all of these changes since the `gecko2` feature flag override isn't necessary since the related PR was reverted.

@jdamore-linode That's good info, thanks for sharing. It's also interesting because I recognized that I was repeating the same operation for each test, so I had consolidated the calls as you described and the tests failed. In any event, I've merged the latest from developed and removed the previous changes as requested.

**Review by jdamore-linode — APPROVED** [review]
Thanks @carrillo-erik! Approved pending CI passing (and I'll follow up if there are failures since there are a couple unrelated flaky tests atm)

> I had consolidated the calls as you described and the tests failed

Hm -- wish I had an explanation for you as to why that didn't work!

**Pull request merged**